expyfun/_experiment_controller.py

```diff
@@ -1069,6 +1069,10 @@ def _log_presses(self, pressed):
     def check_force_quit(self):
         """Check to see if any force quit keys were pressed."""
         self._response_handler.check_force_quit()
+    
+    def check_interrupts(self):
+        """Check to see if any interrupt keys were pressed."""
+        self._response_handler.check_interrupts()
```

Comment:
Shouldn't be part of this PR, right?

Reply:
Yes, that is my mistake

Reply:
(serves me right for trying to multitask)